example/main.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/notaryproject/notation-plugin-framework-go/cli"
+)
+
+func main() {
+	ctx := context.Background()
+	// Initialize plugin
+	plugin, err := NewExamplePlugin()
```

Comment:
NewExamplePlugin() undefined error! Because of main package method cannot be invoked in another main package. Maybe move plugin.go to another package.

Reply:
Its working, verified it locally.